sys/linux/socket_netlink_route.txt

```diff
@@ -879,6 +882,94 @@ rtcanmsg {
 
 rtcanmsg_flags = CGW_FLAGS_CAN_ECHO, CGW_FLAGS_CAN_SRC_TSTAMP, CGW_FLAGS_CAN_IIF_TX_OK
 
+#TODO: define remaining lwtunnel_encap types
+lwtunnel_policy [
+	MPLS_IPTUNNEL_DST	nlattr[MPLS_IPTUNNEL_DST, array[int8, 4]]
```

Comment:
Should this be an array of labels, each being `struct mpls_shim_hdr` or something like that?

Reply:
Per discussion offline, we think that an existing definition of this exists in sys/linux/vnet.txt named 'mpls_label'. That one should probably be used.